<launch>
	<node name="roburoc4_menu" pkg="roburoc4_menu" type="roburoc4_menu" output="screen" />
</launch>
